src/main/app/src/app/zaken/model/besluit-intrekken-gegevens.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {VervalReden} from './vervalReden';
+
+export class BesluitIntrekkenGegevens {
+    besluitUuid: string;
+    vervaldatum: string;
+    vervalreden: VervalReden;
+    reden: string;
```

Comment:
Wat betekend reden? wijzigingsReden?

Reply:
Conform naamgeving in andere gegevens-objecten.